### ISSUE TITLE
[Component] - Audio Player

### ISSUE THREAD

**Issue by alemesa** [Jam3/react-ui#19]

**Comment by jmckinnell** [comment]
@alemesa Were you thinking of something that made use of `Howlerjs` or just a simple HTML5 Audio tag?

**Comment by alemesa** [comment]
yeah, it was using regular audio tag @jmckinnell but not sure what's the standard now, best practices or maybe a library

I did one for complexland 1 https://github.com/Jam3/prj-2-267-complex-complexland-3-frontend/tree/develop/src/components/Radio but is very spicy w/ lots of specific functionality and I feel is better to just do one from scratch
